project/.bloop/support-frontend-build.json

```diff
@@ -0,0 +1,2566 @@
+{
```

Comment:
I don't think you want this!

Reply:
ahhhh idk why that keeps happening atm, I need to investigate